### DISCUSSION TITLE
Issue with go install

### DISCUSSION THREAD

**Discussion by fauns-o** [projectdiscovery/subfinder#504] [Q&A] [answered]
get "gopkg.in/tomb.v1": found meta tag vcs.metaImport{Prefix:"gopkg.in/tomb.v1", VCS:"git", RepoRoot:"https://gopkg.in/tomb.v1"} at //gopkg.in/tomb.v1?go-get=1
get "gopkg.in/fsnotify.v1": found meta tag vcs.metaImport{Prefix:"gopkg.in/fsnotify.v1", VCS:"git", RepoRoot:"https://gopkg.in/fsnotify.v1"} at //gopkg.in/fsnotify.v1?go-get=1
go install github.com/projectdiscovery/subfinder/v2/cmd/subfinder@latest: github.com/projectdiscovery/subfinder/v2@none updating to
	github.com/projectdiscovery/subfinder/v2@v2.4.9 requires
	golang.org/x/net@v0.0.0-20210415231046-e915ea6b2b7d: unrecognized import path "golang.org/x/net": https fetch: Get "https://golang.org/x/net?go-get=1": dial tcp 142.251.42.241:443: connect: connection refused

**Comment by fauns-o** [comment]
go install -v github.com/projectdiscovery/subfinder/v2/cmd/subfinder@latest

go command installed How to install subfinder via go command
The above command does not work

**Comment by ehsandeep** [comment, marked as answer]
@hunter-00 
> dial tcp 142.251.42.241:443: connect: connection refused

that's a network issue on your system.

**Reply by fauns-o** [reply]
谢谢 我已经知道原因了 go代理环境变量 版本不匹配
